File: JS/10-18-25/JS/leap_year2.js
let count = 0

for (let year = 1900; year <= 2025; year++){
	if (year % 100 == 0){
		if (year % 400 == 0)
			count ++;
}
	if (year % 4 == 0 and year % 100 != 0)
		count += 1;

console.log(count);

File: JS/10-18-25/Python/Leap_year2.py
count = 0

for year in range(1000, 2026):
	if year % 100 == 0:
		if year % 400 == 0:
			count += 1
	if year % 4 == 0 and year % 100 != 0:
		count += 1

print(count)